static/css/main.scss

```diff
@@ -37,12 +33,29 @@ body {background-color: $newsprint-tint1;}
     font-family: 'Oranienbaum', serif;
     font-size: 3em; 
     font-variant: small-caps;
+    font-weight: 800;
     margin: 0;
     padding: .2em;
     text-align: center;
   } 
 }
 
+a {
+  text-decoration: underline;
+}
+
+.nav a, .btn a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline !important; //Spent a while trying to figure this out, using this for now
```

Comment:
Guessing that the order of loading multiple CSS files is affecting this such that you had to use an overriding `!important`? Don't want to change this, but curious what's going on that prompted the comment :)

Reply:
I was trying to get links to always underline on hover, but something was not letting me. I suspect that yes, it's the order of the loading - not sure if the bootstrap file is getting loaded after this one or what. It may also be the specificity of the bootstrap file. I'll make an issue to resolve this in the future, but ideally, we could do it in combination with combining the bootstrap and main style files so I don't have to repeat color variables, etc.